### PR TITLE
Feature forms

### PR DESCRIPTION
**Pull request by mikethreels**
# Milestone 1: Created the Reformer.

## In this milestone we accomplished:

- [x] implemented the User model, controller, and view including validations in the model
- [x] Implemented plain HTML form, updating it to a form_tag which was later updated to for_form
- [x] added the methods for creating and updating users
- [x] Implemented the error display in the browser.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_user_path
+    else
+      render new_user_path
```
Comment (mosesogwo) [inline app/controllers/users_controller.rb]: `render :new` is more appropriate here
You can read about their difference [here](https://medium.com/@kerenlerner/render-and-redirect-which-to-use-106ff653ee9a)

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello! Great job :clap: Something is still missing in the project**

:point_right: I can't find your commented out forms - the ones created with **html form element** and **form_tag helpers**






_**Please include them and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by santiagorodriguezbermudez** [comment]
# Changes # 1

## On this update we change the following:

- [x] Added the html as a comment code to the new and edit views.
- [x] Added the form_tag helper form as a comment code to the new and edit views. 
- [x] Used the :new method for the controller as suggested per the TSE

**Review by uche-inyama — APPROVED** [review]
## Approved

Hi team :wave: 

this project is approved, go on with the next project. :100: 

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**